### ISSUE TITLE
CPU-only mode in notebook after model_convert

### ISSUE THREAD

**Issue by Alexankharin** [fizyr/keras-retinanet#1041]
during model conversion at convert_model.py we got the os pass variable created
os.environ["CUDA_VISIBLE_DEVICES"] = ""
and start keras on CPU
in case of library usage in jupyter-notebook project, i.e

import keras_retinanet
from keras_retinanet import preprocessing
from keras_retinanet.preprocessing import csv_generator
from keras_retinanet import models as mdls
from keras_retinanet.bin import convert_model
model=mdls.backbone('resnet50').retinanet(num_classes=len(classes))
model.compile(
    loss={
        'regression'    : keras_retinanet.losses.smooth_l1(),
        'classification': keras_retinanet.losses.focal()
    },
    optimizer=keras.optimizers.adam(lr=1e-5, clipnorm=0.001)
)
model.fit_generator(PARAMS)
model.save(PASS)
convert_model.main([PASS,INFERPASS])
model=mdls.load_model(INFERPASS)
the inference model runs on CPU-only mode, so it is good write the code for correcting os.environ["CUDA_VISIBLE_DEVICES"] after conversion and restarting the tf session in GPU mode

**Comment by hgaiser** [comment]
Hmm yes I see what you mean. Could you make a PR to change the setting of that environment variable to the `if __name__ == '__main__':` block? That way it is only called when it is run in a standalone way.

**Comment by hgaiser** [comment]
Any update on this?
